Makefile

```diff
@@ -0,0 +1,5 @@
+load-smoke-test:
```

Comment:
I think having consistent make task names would be cool here, e.g. `make smoke-tests` and `make browser-tests`. Maybe even `make django-tests`?